### ISSUE TITLE
Get the specified metrics because all BucketMetrics are too large

### ISSUE THREAD

**Issue by nowinkeyy** [minio/madmin-go#275]
The metricsRespBodyLimit size is 10MB, but the actual data is 14MB>10MB, causing an error. Can I specify a metric to access? For example, minio_bucket_usage_total_bytes metric.
```
const (
	metricsRespBodyLimit = 10 << 20 // 10 MiB
)
// fetchMetrics - returns Metrics of given subsystem in Prometheus format
func (client *MetricsClient) fetchMetrics(ctx context.Context, subSystem string) ([]*prom2json.Family, error) {
	reqData := metricsRequestData{
		relativePath: "/v2/metrics/" + subSystem,
	}

	// Execute GET on /minio/v2/metrics/<subSys>
	resp, err := client.executeGetRequest(ctx, reqData)
	if err != nil {
		return nil, err
	}
	defer closeResponse(resp)

	if resp.StatusCode != http.StatusOK {
		return nil, httpRespToErrorResponse(resp)
	}

	return parsePrometheusResults(io.LimitReader(resp.Body, metricsRespBodyLimit))
}
```

**Comment by nowinkeyy** [comment]
I'm not sure what the 10MB limit means here. My current workaround is to copy some of the code and filter out the metric information before parsePrometheusResults. After filtering, the data goes from 14MB to 72KB so that it doesn't exceed 10MB.

**Comment by harshavardhana** [comment]
This is addressed in #276
